app/main.py

```diff
@@ -1,2 +1,48 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8, load_factor: float = 0.66) -> None:
+        self.capacity = capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self.hash_table = [None] * 8
+
+    def __setitem__(self, key: Any, value: Any) -> None:
```

Comment:
Are you sure key could be any type? What about None, list? Key should have **hashable** type